.drone.star

```diff
@@ -858,7 +856,8 @@ def acceptance(ctx):
 	errorFound = False
 
 	default = {
-		'servers': ['daily-master-qa', 'latest'],
+		#'servers': ['daily-master-qa', 'latest'],
+		'servers': ['daily-master-qa'],
```

Comment:
I normally prefer not to change the "standard" code below `def main`. Because this code is the same in all apps, and we regularly copy the "latest and best"  code starlark changes into all the apps.

Should be OK for today. We should be bale to revert this in 1 week, when `latest` points to a 10.8 release tarball.

Reply:
:+1: